pallets/attestation/src/lib.rs

```diff
@@ -253,25 +252,118 @@ pub mod pallet {
 			let source = <T as Config>::EnsureOrigin::ensure_origin(origin)?;
 			let payer = source.sender();
 			let who = source.subject();
+
```

Comment:
Yes, you are right. Nevertheless, the new pallet depends on these features to be available. Would I have to be a new PR for that and rebase this one on top of that? I thought it would be logical to do it here.